src/gwt/src/org/rstudio/core/client/VirtualConsole.java

```diff
@@ -29,7 +29,6 @@
 import org.rstudio.core.client.regex.Match;
 import org.rstudio.core.client.regex.Pattern;
 import org.rstudio.core.client.virtualscroller.VirtualScrollerManager;
-import org.rstudio.studio.client.RStudioGinjector;
```

Comment:
Can we add some unit tests for this scenario? The ANSI / Console stuff handling code is fragile and complex so having lots of coverage is a good idea. Lots of examples in: https://github.com/rstudio/rstudio/blob/main/src/gwt/test/org/rstudio/core/client/VirtualConsoleTests.java

Reply:
I added two, but I currently struggle with my java settings ...: 

```
romainfrancois@mezcalita gwt % ./ant unittest
Building with Java at /opt/homebrew/opt/openjdk@11
Using JAVA_HOME: /opt/homebrew/opt/openjdk@11
Buildfile: /Users/romainfrancois/git/rstudio/src/gwt/build.xml

build-unittests:

unittest:
     [java] JUnit version 4.9b3
     [java] Exception in thread "main" java.lang.UnsupportedClassVersionError: org/rstudio/studio/client/RStudioUnitTestSuite has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0
     [java] 	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
     [java] 	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1017)
     [java] 	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
     [java] 	at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:800)
     [java] 	at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:698)
     [java] 	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:621)
     [java] 	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:579)
     [java] 	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
     [java] 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
     [java] 	at java.base/java.lang.Class.forName0(Native Method)
     [java] 	at java.base/java.lang.Class.forName(Class.java:315)
     [java] 	at org.junit.runner.JUnitCore.runMain(JUnitCore.java:89)
     [java] 	at org.junit.runner.JUnitCore.runMainAndExit(JUnitCore.java:53)
     [java] 	at org.junit.runner.JUnitCore.main(JUnitCore.java:45)

BUILD FAILED
/Users/romainfrancois/git/rstudio/src/gwt/build.xml:358: Java returned: 1

Total time: 0 seconds
```

Reply:
Do you have any luck with `./ant clean && ./ant unittest`? If not, we might need to tweak how JAVA_HOME is found and set in the `ant` helper script.

Reply:
`./ant clean` typically helps for `./ant devmode` but for some reason not for `./ant unittest`. I'll investigate.